containers/filer.Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN python3 setup.py bdist_wheel
 
 FROM alpine:3.10
 
-RUN apk add --no-cache python3
+RUN apk add --no-cache python3 gcc libffi-dev openssl-dev python3-dev libc-dev
```

Comment:
Why exactly do we need those dependencies?

Reply:
removed it as it was not necessary